src/App.js

```diff
@@ -9,20 +9,20 @@ function App() {
   const [isLoading, setIsLoading] = useState(true)
 
   useEffect(() => {
-    new Promise((resolve, reject) => {
-      setTimeout(() => {
-        resolve({
-          data: {
-            todoList: JSON.parse(localStorage.getItem(initialTodoList)) || []
-          }
-        })
-      }, 2000);
-    })
-      .then(({ data }) => {
-        setTodoList(data.todoList);
+    fetch('https://api.airtable.com/v0/app6WkPQtNjU0dxxU/Default',
+      {
+        headers: { "Authorization": "Bearer key889ohx5Mc5ygVh" }
```

Comment:
You should hide your airtable `Base ID` and `API KE`Y as in instruction to lesson-1-8 https://github.com/Code-the-Dream-School/ctd-react-canary/wiki/Lesson-1.8. But I think instruction is not too clear. Do these steps:

1. in the root directory of your project create file `.env.local` and put inside these two lines. (Important: each line MUST start with **REACT_APP_** )
```
REACT_APP_BASE_ID=app6WkPQtNjU0dxxU
REACT_APP_API_KEY=key889ohx5Mc5ygVh
```
2. Then update your fetch request with 
```
fetch(`https://api.airtable.com/v0/${process.env.REACT_APP_BASE_ID}/Default`,
      {
        headers: { "Authorization": `Bearer ${process.env.REACT_APP_API_KEY}` }
      })
```

This should work! Let me know if it's not.